kontonummer.js

```diff
@@ -448,15 +437,25 @@
 
         return {
             errors: errors,
-            bankName: bankName,
-            clearing_number: getClearingNumber(bankNumber, bank),
-            account_number: getAccountNumber(bankNumber, bank)
+            bank_name: bankName,
+            clearing_number: getClearingNumber(bankNumber, bank.lengths.clearing),
+            account_number: getAccountNumber(bankNumber, bank.lengths.clearing)
```

Comment:
I can't comment on the right line, but regarding `snake_case` here, do we want to be consistent with `isValid` and `matchedBanks` returned in the constructor on line `400` as well that are camelCased now? Seems a bit weird to return `camelCase` properties with an array of objects with `snake_case` properties.

Don't forget to change the `README` file as well.